12-offer-encoding.md

```diff
@@ -238,296 +233,328 @@ The human-readable prefix for offers is `lno`.
 ## Requirements For Offers
 
 A writer of an offer:
-  - if it sets `node_id`:
-    - MUST set `node_id` to the node's public key to request the invoice from.
-  - otherwise:
-    - MUST provide at least one `blinded_path`
-    - MUST use the final `onionmsg_path` `point` in the first `blinded_path` as the implied `node_id` for `signature`.
-  - MAY specify exactly one signature TLV: `signature`:
-    - If so, it MUST set `sig` to the signature using `node_id` as described in [Signature Calculation](#signature-calculation).
-  - MUST set `description` to a complete description of the purpose
+  - MUST NOT set any tlv fields greater or equal to 80, or tlv field 0.
```

Comment:
I think it's worth adding a high level explanation somewhere explaining why we limit the range of TLV fields? 

We still specify `tlv_stream: offer/invoice_request/invoice` as separate streams, but they're actually related which is different to other TLV streams in the spec.